rasterio/_crs.pyx

```diff
@@ -28,6 +28,48 @@ def gdal_version():
     return info_b.decode("utf-8")
 
 
+def _epsg_treats_as_latlong(input_crs):
+    """Test if the CRS is in latlon order
+
+    Parameters
+    ----------
+    input_crs : CRS
+        rasterio CRS object
+
+    Returns
+    -------
+    bool
+
+    """
+    cdef _CRS crs = input_crs._crs
```

Comment:
@sgillies this feels kind of strange, not sure what is the best way to access the _CRS object

Reply:
looking at https://github.com/mapbox/rasterio/blob/master/rasterio/crs.py#L92 I think should pass `crs._crs` to `_epsg_treats_as_northingeasting` form `crs.epsg_treats_as_northingeasting`